charts-example/src/app/extensions/ChartsExampleCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Text } from '@hubspot/ui-extensions';
+import { LineChart, BarChart } from '@hubspot/ui-extensions/experimental';
```

Comment:
The charts imports are no longer in experimental. They are available at `@hubspot/ui-extensions`